templates/bridgeserver2.yaml

```diff
@@ -759,6 +759,83 @@ Resources:
       Period: 300
       Statistic: Average
       Threshold: 1
+  # Bucket for raw health data. Grant Synapse read-only permissions to this bucket. Only Bridge can
+  # write to thise bucket.
+  RawHealthDataBucket:
```

Comment:
If these buckets are external dependencies of the server app then I would suggest decoupling them by putting them in a separate CFN file.

Reply:
No, these are created and used only by the BridgeServer. Actually, I need to add the env vars to make them available to Elastic Beanstalk environment.